src/config.rs

```diff
@@ -98,6 +99,8 @@ const NEON_HEAP_FRAME: &str = "NEON_HEAP_FRAME";
 const NEON_ADDITIONAL_FEE: &str = "NEON_ADDITIONAL_FEE";
 const NEON_OPERATOR_KEYFILE: &str = "NEON_OPERATOR_KEYFILE";
 const NEON_ETH_MAX_AMOUNT: &str = "NEON_ETH_MAX_AMOUNT";
+const NEON_ETH_PER_TIME_MAX_AMOUNT: &str = "NEON_ETH_PER_TIME_MAX_AMOUNT";
+const NEON_ETH_SLICE: &str = "NEON_ETH_SLICE";
```

Comment:
Can we call this `NEON_ETH_TIME_SLICE_SECS`?

Reply:
was trying to stick with solana naming convention but sure will change
https://github.com/Eclipse-Laboratories-Inc/solar-eclipse/blob/cd4a6952b76b65eec90f09cfc3b68066e2fda7d0/faucet/src/bin/faucet.rs#L38